pubs/commands/import_cmd.py

```diff
@@ -8,28 +8,32 @@
 from .. import bibstruct
 from .. import color
 from ..paper import Paper
-
 from ..uis import get_ui
 from ..content import system_path, read_text_file
+from ..command_utils import add_doc_add_arguments
 
 
 _ABORT_USE_IGNORE_MSG = "Aborting import. Use --ignore-malformed to ignore."
 _IGNORING_MSG = " Ignoring it."
 
 
 def parser(subparsers, conf):
-    parser = subparsers.add_parser('import',
-            help='import paper(s) to the repository')
-    parser.add_argument('bibpath',
-            help='path to bibtex, bibtexml or bibyaml file (or directory)')
-    parser.add_argument('-L', '--link', action='store_false', dest='copy', default=True,
-            help="don't copy document files, just create a link.")
-    parser.add_argument('keys', nargs='*',
-            help="one or several keys to import from the file")
-    parser.add_argument('-O', '--overwrite', action='store_true', default=False,
-            help="Overwrite keys already in the database")
-    parser.add_argument('-i', '--ignore-malformed', action='store_true', default=False,
-            help="Ignore malformed and unreadable files and entries")
+    parser = subparsers.add_parser(
+        'import',
+        help='import paper(s) to the repository')
+    parser.add_argument(
+        'bibpath',
+        help='path to bibtex, bibtexml or bibyaml file (or directory)')
+    parser.add_argument(
+        'keys', nargs='*',
+        help="one or several keys to import from the file")
+    parser.add_argument(
+        '-O', '--overwrite', action='store_true', default=False,
+        help="Overwrite keys already in the database")
+    parser.add_argument(
+        '-i', '--ignore-malformed', action='store_true', default=False,
+        help="Ignore malformed and unreadable files and entries")
+    add_doc_add_arguments(parser, move=False)
```

Comment:
What the rational for not allowing move here? A comment in the code could be helpful.

Reply:
Well,
– the not-so-convincing reason was to match the current behavior,
– the real reason was that I was not confortable having a default behavior that moves files because of some mention in the bibtex that is not clearly visible from the user. More precisely having some `file = "..."` in the imported bibtex would mean having these files moved around. Since this is not as clear as in the `add` command in which the user explicitly passes a `--doc` argument.

I believe nonetheless that it would be good to have a move command line argument for the `import` command but it should probably not share the config option with the `add` command and its default. (This somehow makes sense since the config option is called `doc_add`.) We could eventually add a config option called `doc_import` to make it event clearer.

As a matter of fact, I am also not sure that `doc_add = 'move'` is the right default and I would rather have a default to `copy`.

Any opinion on that?

Reply:
I see. I don't believe that a `doc_import` is needed. We don't import regularly. We should just support a command line argument down the line.
I think `copy` as a default is a good idea.